runtime/interpreter/value_accountcapabilitycontroller.go

```diff
@@ -195,13 +196,15 @@ func (v *AccountCapabilityControllerValue) GetMember(inter *Interpreter, _ Locat
 
 	switch name {
 	case sema.AccountCapabilityControllerTypeTagFieldName:
-		if v.tag == nil {
+		if v.tag == nil || v.tag.graphemes == nil {
 			v.tag = v.GetTag()
 			if v.tag == nil {
 				v.tag = EmptyString
 			}
 		}
 		return v.tag
+	case sema.AccountCapabilityControllerTypeSetTagFunctionName:
+		return v.SetTagFunction
```

Comment:
Maybe make `SetTagFunction` unexported, `NewAccountCapabilityControllerSetTagFunction` unexported and a receiver method of the controller type, `newSetTagFunction`, and just initialize here lazily:
```suggestion
		if v.setTagFunction == nil {
			v.setTagFunction = v.newSetTagFunction(inter)
		}
		return v.setTagFunction
```

Then, the initialization in `stdlib/account.go` is not necessary (avoids it from being forgotten)